types/uniqid/index.d.ts

```diff
@@ -4,7 +4,7 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
```

Comment:
Can you correct the header to attribute yourself as the author?

Reply:
What do you mean? I'm not the original author of uniqid types. Just trying to fix the thing 😄